spring-cloud-dataflow-common/pom.xml

```diff
@@ -50,19 +44,6 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
-	<build>
```

Comment:
This is all specified in parent builds. I wanted to remove it here first in a small slice and if all goes well move to the rest of SCDF in a separate subsequent code proposal.